iocage_lib/ioc_start.py

```diff
@@ -586,6 +540,59 @@ def __start_jail__(self):
                 _callback=self.callback,
                 silent=self.silent)
 
+            if not vnet_err and vnet and wants_dhcp:
+                failed_dhcp = False
+
+                try:
+                    interface = self.conf["interfaces"].split(",")[0].split(
+                        ":")[0]
```

Comment:
You are not making sense to my OCD. You're switching a bunch of double quotes to single quotes in a lot of PRs. And in this new entire code block you're using double quotes? :P